fe/fe-core/src/main/java/com/starrocks/connector/delta/FileScanTask.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector.delta;
+
+import io.delta.kernel.utils.FileStatus;
+
+import java.util.Map;
+
+public class FileScanTask {
```

Comment:
It is better to add some comments for this class.

Reply:
Is it right that each scan task will contain only one file?
If yes, why not let one task have multiple files?

Reply:
This just represents a file. There is a problem with the naming. It may be more reasonable to use FileInfo. It is mainly consistent with the implementation of Iceberg.

Reply:
Later I will think of a unified name and abstract a parent class.

Reply:
Add comment here, will fixed later